.jenkins/pytorch/perf_test/common.sh

```diff
@@ -10,7 +10,7 @@ get_runtime_of_command () {
   TIMEFORMAT=%R
 
   # runtime=$( { time ($@ &> /dev/null); } 2>&1 1>/dev/null)
-  runtime=$( { time $@; } 2>&1 1>/dev/null)
+  runtime=$( { time "$@"; } 2>&1 1>/dev/null)
```

Comment:
Is this always right too? I don't know how this is used, but if we wanted to pass commands and arguments to that command wouldn't we want $@ to be reinterpreted?